### PR TITLE
[next] fix: Adjust tsconfig and eslintrc for tests

### PR DESCRIPTION
**Pull request by susnux**
### ☑️ Resolves

* Fixes `rootdir` in invalid place in tsconfig.
* Fixes wrong location of eslintrc

### 🏁 Checklist

- [x] ⛑️ Tests are included or are not applicable
- [x] 📘 Component documentation has been extended, updated or is not applicable

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -12,11 +12,13 @@
     "strict": true,
     "noImplicitAny": false,
     "outDir": "./dist",
+    "rootDir": "./src",
```
Comment (ShGKme) [inline tsconfig.json:15]: Maybe make it just the root to keep structure?
```suggestion
    "rootDir": ".",
```

**Review by ShGKme — COMMENTED** [review]
Is it needed only for `next`?

File: tsconfig.json (hunk 1 above)
Comment (susnux) [reply tsconfig.json:15]: The the output will be `dist/src/...` instead of `dist/...`

**Comment by susnux** [comment]
> Is it needed only for next?

Yes only on next this is an issue

**Review by ShGKme — APPROVED** [review]

**Review by Antreesy — APPROVED** [review]

**Pull request merged**